docs/tutorials/iers.rst

```diff
@@ -0,0 +1,135 @@
+:orphan:
+
+.. doctest-skip-all
+
+**********************************
+Using and updating IERS Bulletin A
+**********************************
+
+Contents
+========
+
+* :ref:`iers-Background`
+* :ref:`iers-What_are_IERS_Bulletins_A_and_B`
+* :ref:`iers-How_do_I_get_IERS_Bulletin_A_for_astroplan?`
+
+.. _iers-Background:
+
+Background
+==========
+
+Keeping accurate, consistent time systems is rough. We intuitively want
+time to be described by a continuous cyclic clock like the clock on your
+wall or the calendar, but the Earth's rate of rotation varies measurably
+on human timescales. As a result, the time measured by an atomic clock and
+the time that you would infer from measuring how long it has been since the
+last solar noon are constantly becoming offset from one another with
+stochastic changes in Earth's moment of inertia and slowly acting tidal forces.
+
+For this reason, there is a time system that keeps track of seconds like an
+atomic clock, UTC, and one that matches up with the Earth's rotation, UT1. The
+difference between UT1 and UTC is constantly changing as the Earth's rotation
+changes and as leap seconds get added to UTC to compensate.
```

Comment:
I know you're trying to be pedagogical here and don't want to get into the details here: that's very good! This is easy-to-read, which is probably going to be good for astroplan users.

But I think you need to make a modification here to make it clear that UTC is *not* just a continuous progression of atomic clock time - that's what TAI is.  UTC is that plus a constant offset that changes every so often to make it stay within a second of UT1.  This isn't just pedantic - it's important for users to know that any leap seconds that get added in between now and whatever time they're thinking about do *not* get accounted for because we can't accurately predict leap seconds.